### PR TITLE
feat(about): added about section cards

### PR DESCRIPTION
**Pull request by z3r0n3br4instorm**
bruh

### CODE REVIEW

File: src/components/dataCard.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        <div
+          style={{
+            flex: 1,
+            display: "flex",
+            justifyContent: "center",
+            alignItems: "center",
+            border: "0px solid #ccc",
+          }}
+        >
+          <img
```
Comment (z3r0n3br4instorm) [inline src/components/dataCard.tsx:49]: Gamora, I'm Sorry !

**Review by AyoItsYas — CHANGES_REQUESTED** [review]
About page is missing, create the page.

File: src/app/page.tsx
Comment (AyoItsYas) [inline src/app/page.tsx:1]: Please create a new page for the contact page. Refer: https://nextjs.org/docs/app/building-your-application/routing

File: src/components/dataCard.tsx
Comment (AyoItsYas) [inline src/components/dataCard.tsx:1]: Anything can be called data rename the components to a sensible name, use ChatGPT, and with the follow kebab case file naming. `dataCard` should be `data-card`.

File: src/components/dataCard.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+*/
+
+export default function DataCard({ data }: { data: string }) {
+  // Split text to 2 parts from | character
+  const [para, imgsrc] = data.split("|");
+
+  return (
+    <div
+      style={{
+        width: "1200px",
```
Comment (AyoItsYas) [inline src/components/dataCard.tsx]: Refactor these inline styles to use tailwind, or without repeating the same set of lines create the styles object out inside the component scope without repeating the same lines.

File: src/components/topLogo.tsx
Comment (AyoItsYas) [inline src/components/topLogo.tsx:1]: Again bad file naming.

**Review by AyoItsYas — CHANGES_REQUESTED** [review]

File: src/components/dataCard.tsx (hunk 2 above)
Comment (z3r0n3br4instorm) [reply src/components/dataCard.tsx]: ah shit mek fix krnn beriuna 
Fs in the chat, hdnnm gimme a sec

Comment (z3r0n3br4instorm) [reply src/components/dataCard.tsx]: Done

**Review by AyoItsYas — APPROVED** [review]

**Pull request merged**